sdk/vaa/payloads.go

```diff
@@ -73,47 +83,39 @@ func (b BodyGuardianSetUpdate) Serialize() []byte {
 }
 
 func (r BodyTokenBridgeRegisterChain) Serialize() []byte {
-	if len(r.Module) > 32 {
-		panic("module longer than 32 byte")
-	}
+	return serializeBridgeGovernanceVaa(r.Module, 1, r.ChainID, r.EmitterAddress)
+}
 
-	buf := &bytes.Buffer{}
+func (r BodyTokenBridgeUpgradeContract) Serialize() []byte {
+	return serializeBridgeGovernanceVaa(r.Module, 2, r.TargetChainID, r.NewContract)
+}
 
-	// Write token bridge header
-	for i := 0; i < (32 - len(r.Module)); i++ {
-		buf.WriteByte(0x00)
-	}
-	buf.Write([]byte(r.Module))
-	// Write action ID
-	MustWrite(buf, binary.BigEndian, uint8(1))
-	// Write target chain (0 = universal)
-	MustWrite(buf, binary.BigEndian, uint16(0))
-	// Write chain to be registered
-	MustWrite(buf, binary.BigEndian, r.ChainID)
-	// Write emitter address of chain to be registered
-	buf.Write(r.EmitterAddress[:])
+func (r BodyWormchainStoreCode) Serialize() []byte {
+	return serializeBridgeGovernanceVaa("WasmdModule", 1, ChainIDWormchain, r.WasmHash)
```

Comment:
Are these action IDs documented somewhere? Should they be consts?

Reply:
The action IDs and the "WasmdModule" string are currently defined in the wormchain `msg_server_wasmd` - https://github.com/wormhole-foundation/wormhole/blob/main/wormchain/x/wormhole/keeper/msg_server_wasmd.go#L15-L21

Maybe worth moving them all over to the sdk?